vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -99,12 +99,15 @@ def build(self):
 
     @staticmethod
     def _update_theme():
-        @callback(
+        clientside_callback(
```

Comment:
Same question on what the advantage of doing this is

Reply:
It seems strange to do a calculation like:
```
if on == True:
      return 'vizro_dark'
else:
      return 'vizro_light'
```
on the server side. This means that if users change the theme, they have to wait about 400 ms for a response. By moving the callback to the client side, this change happens instantly.

Ideally we should define callbacks on the server-side only if they have to communicate with data_manager, model_manager or some other service that's hosted on the server together with the vizro app.